package.json

```diff
@@ -3,12 +3,25 @@
   "version": "1.0.1",
   "description": "Solid-compatible Panes: File table",
   "main": "./folderTable.js",
+  "files": [
+    "src",
+    "lib"
+  ],
   "scripts": {
+    "build": "webpack",
+    "build-browserified": "npm run build",
+    "build-dist": "webpack --progress --colors --mode=production",
+    "build-dev": "webpack --progress --colors --mode=development",
+    "build-lib": "babel src -d lib --source-maps --extensions '.ts,.js'",
+    "build-types": "tsc --emitDeclarationOnly",
+    "build-version": "./timestamp.sh > src/versionInfo.ts  && eslint 'src/versionInfo.ts' --fix",
     "lint": "eslint '*.js'",
     "lint-fix": "eslint '*.js' --fix",
     "test": "npm run lint",
     "prepublishOnly": "npm test",
-    "postpublish": "git push origin master --follow-tags"
+    "postpublish": "git push origin master --follow-tags",
+    "watch": "webpack --watch",
```

Comment:
Maybe we should distinguish between watch for the webapp, and watch for the libs? E.g. could have `app-watch` and `lib-watch`?

Reply:
Renamed it. Thanks!